credentials/alts/alts_test.go

```diff
@@ -300,3 +353,49 @@ func versions(minMajor, minMinor, maxMajor, maxMinor uint32) *altspb.RpcProtocol
 		MaxRpcVersion: version(maxMajor, maxMinor),
 	}
 }
+
+func startFakeHandshakerService(t *testing.T, wait *sync.WaitGroup) (stop func(), address string) {
+	listener, err := net.Listen("tcp", ":0")
```

Comment:
We have a testutil to create a local TCP listener which also works well in forge. See: https://github.com/grpc/grpc-go/blob/master/internal/testutils/local_listener.go#L24

Reply:
Oh awesome! Thanks for pointing this out!